pkg/apis/core/v1/conversion.go

```diff
@@ -539,3 +546,146 @@ func Convert_core_PersistentVolumeSpec_To_v1_PersistentVolumeSpec(in *core.Persi
 func Convert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in *v1.PersistentVolumeSpec, out *core.PersistentVolumeSpec, s conversion.Scope) error {
 	return autoConvert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in, out, s)
 }
+
+// HandleSeccompVersionSkew implements the version skew behavior described in:
+// https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190717-seccomp-ga.md#version-skew-strategy
+func HandleSeccompVersionSkew(in *core.Pod, out *v1.Pod) error {
+	// Handle the pod
+	annotationValue := in.Annotations[v1.SeccompPodAnnotationKey]
+	var podInField *core.SeccompProfile
+	if in.Spec.SecurityContext != nil {
+		podInField = in.Spec.SecurityContext.SeccompProfile
+	}
+
+	profile, err := syncSeccompAnnotationAndFieldForPod(
+		v1.SeccompPodAnnotationKey, annotationValue, podInField, out,
+	)
+	if err != nil {
+		return errors.Wrapf(err, "synchronizing annotations and security context field for pod: %+v", in)
+	}
+
+	if profile != nil {
+		// Ensure that the output security context is not nil and set the new
+		// profile
+		if out.Spec.SecurityContext == nil {
+			out.Spec.SecurityContext = &v1.PodSecurityContext{}
+		}
+		out.Spec.SecurityContext.SeccompProfile = profile
+	}
+
+	// Handle the containers of the pod
+	for i, ctr := range in.Spec.Containers {
```

Comment:
Note: Use `podutil.VisitContainers` like in https://github.com/kubernetes/kubernetes/pull/91381/files#diff-81c0c668d90c62bb5b7a2334c09c7e50R3781